lib/mpl_toolkits/mplot3d/axis3d.py

```diff
@@ -83,11 +84,18 @@ def __init__(self, adir, v_intervalx, d_intervalx, axes, *args, **kwargs):
         self._axinfo.update({'label' : {'va': 'center',
                                         'ha': 'center'},
                              'tick' : {'inward_factor': 0.2,
-                                       'outward_factor': 0.1},
-                             'axisline': {'linewidth': 0.75,
-                                          'color': (0, 0, 0, 1)},
-                             'grid' : {'color': (0.9, 0.9, 0.9, 1),
-                                       'linewidth': 1.0},
+                                       'outward_factor': 0.1,
+                                       'linewidth': rcParams.get(
+                                           adir + 'tick.major.width',
+                                           rcParams['xtick.major.width']),
+                                       'color': rcParams.get(
+                                           adir + 'tick.color',
+                                           rcParams['xtick.color'])},
+                             'axisline': {'linewidth': rcParams['axes.linewidth'],
+                                          'color': rcParams['axes.edgecolor']},
+                             'grid' : {'color': rcParams['grid.color'],
+                                       'linewidth': rcParams['grid.linewidth'],
+                                       'linestyle': rcParams['grid.linestyle']},
```

Comment:
We might need to do a "classic mode" shim here. In v1.5, the grid lines used the linestyle, linewidth, and color that were the defaults for lines, not the defaults for grids. We might need a bunch of ternary statements or just a single if-statement populating this update dictionary. Any new entries should fall back to `None` so that the old behavior is preserved.